packages/gamut/src/Form/FormGroupLabel.tsx

```diff
@@ -28,13 +30,10 @@ export type FormGroupLabelProps = HTMLAttributes<HTMLDivElement> &
     size?: 'small' | 'large';
   };
 
-type disabledLabelStyleProps = {
-  disabled?: boolean;
-};
-
 const labelSizeVariants = variant({
   defaultVariant: 'small',
   prop: 'size',
+  ...formBaseStyles,
```

Comment:
Shared styles need to be on the base key :) 
```suggestion
   base: formBaseStyles,
```